### PR TITLE
NONE: add eventloop-lag

### PR DESCRIPTION
**Pull request by bgvozdev**
**What's in this PR?**
Emitting metrics about evenloop lag

**Why**
Trying to understand wtf is going on with web nodes and why the latency became a problem

Tested in stg

**Added feature flags**
🤠

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -13,6 +16,10 @@ const start = async () => {
 	app.listen(port, () => {
 		getLogger("frontend-app").info(`started at port ${port}`);
 	});
+	const lag = createLag(1000);
+	setInterval(() => {
+		statsd.histogram(metricLag.lagHist, lag(), { }, { });
```
Comment (krazziekay) [inline src/main.ts:21]: Do we have any attributes to identify this 1 sec delay? Maybe pass some extra params just in case.

Comment (bgvozdev) [reply src/main.ts:21]: Sorry @krazziekay not sure I understand... 1 sec is just a frequency how often we want to send this metric to sfx...

**Review by krazziekay — APPROVED** [review]

File: src/main.ts (hunk 1 above)
Comment (krazziekay) [reply src/main.ts:21]: Approved then, its just for testing anyways right?

Comment (bgvozdev) [reply src/main.ts:21]: Yeah, it doesn't affect prod. Just to make sure node.js' event loop is not lagging behind (or to identify that this is the problem...)

**Pull request merged**